docs/db/changelog/changesets/powerauth-java-server/1.8.x/20240704-callback-event-table.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="powerauth-java-server/1.8.x/20240704-callback-event-table.xml" author="Jan Pesek">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="pa_application_callback_event" />
+            </not>
+        </preConditions>
+        <comment>Create a new table pa_callback_event</comment>
+        <createTable tableName="pa_application_callback_event">
+            <column name="id" type="varchar(36)">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="application_callback_id" type="varchar(37)">
+                <constraints nullable="false" foreignKeyName="pa_application_callback_id_fk" referencedTableName="pa_application_callback" referencedColumnNames="id" />
+            </column>
+            <column name="callback_data" type="text">
+                <constraints nullable="false" />
+            </column>
+            <column name="status" type="varchar(255)" >
```

Comment:
Do we really need 255 chars? I think the values will be much shorter.

Reply:
reduced to 32